### PR TITLE
remove crypto validation except .zil

### PR DESCRIPTION
**Pull request by JohnnyJumper**

### CODE REVIEW

**Comment by DeRain** [comment]
@JohnnyJumper Please be sure that you are creating PR's based on UNS branch, not on master

File: Sources/UnstoppableDomainsResolution/NamingServices/CNS.swift [hunk 1]
```diff
@@ -42,7 +42,7 @@ internal class CNS: CommonNamingService, NamingService {
     }
 
     func isSupported(domain: String) -> Bool {
-        return domain.hasSuffix(Self.specificDomain)
+        return !domain.hasSuffix(".zil")
```
Comment (DeRain) [inline Sources/UnstoppableDomainsResolution/NamingServices/CNS.swift:45]: Do we need to keep Self.specificDomain constant?

**Review by DeRain — CHANGES_REQUESTED** [review]

File: Sources/UnstoppableDomainsResolution/NamingServices/CNS.swift (hunk 1 above)
Comment (JohnnyJumper) [reply Sources/UnstoppableDomainsResolution/NamingServices/CNS.swift:45]: not really, I will remove it

**Review by DeRain — APPROVED** [review]

**Review by merenkoff — APPROVED** [review]
LGMT. Good work

**Pull request merged**